pghoard/rohmu/object_storage/s3.py

```diff
@@ -147,19 +152,21 @@ def store_file_from_disk(self, key, filepath, metadata=None, multipart=None):
         size = os.path.getsize(filepath)
         key = self.format_key_for_backend(key)
         metadata = self.sanitize_metadata(metadata)
-        if not multipart or size <= self.multipart_chunk_size:
+        if not multipart and size <= self.multipart_chunk_size:
```

Comment:
I found the current semantics a bit confusing, since passing `multipart=True` might use a multipart upload, but also might not. But on second thought it's probably safer to leave as is.